scripts/install/install_unix.sh

```diff
@@ -35,33 +35,40 @@ SCRIPT_NAME="$0"
 INDENT_WIDTH='  '
 indent=""
 
+IS_INTERACTIVE="false"
+case "$-" in
+  *i*) IS_INTERACTIVE="true" ;;
+esac
+
```

Comment:
This seems to always be false in the environment I'm testing in 🤔 

For reference, I'm testing in a fresh Ubuntu Server 24.04 install. The exiting with status code 2 issue is fixed, but we don't get the coloring like we should.